src/sentry/static/sentry/app/views/eventsV2/table/index.tsx

```diff
@@ -95,6 +97,13 @@ class Table extends React.PureComponent<TableProps, TableState> {
           return;
```

Comment:
@wmak add a `metric.measure()` here to indicate a request was dropped.

Reply:
The request being dropped here would be from a different request being started right? I don't think we want that measured by the SLO since the aim is to see whether we're returning valid results quickly.

Reply:
@wmak i think it's worth doing it. Would you be able to add a marker in the `data` payload to indicate it's stale?

In addition, there would need to be a way to clean up marks: https://github.com/getsentry/sentry/blob/ef82b253b9dc1ec7383e537455ea6cb591f014fe/src/sentry/static/sentry/app/utils/analytics.tsx#L154-L156

Reply:
By not cleaning up the marks, some events may appear taking longer than necessary.

Reply:
@dashed What would be the action we want to take if we see users stating new requests often? I might be misunderstanding what that means, but it seems like valid behaviour. My understanding is we want these SLOs to be actionable. eg. Queries are slow, Queries are erroring often

But I agree on needing to clear up the marks since the name is currently the query :+1:

Reply:
My understanding of this SLO metric is that we should only care about API responses; independent of the frontend. That is, how long it took, and what kind of HTTP responses was received.

Not whether or not they're the right data that should be shown in the UI.

Reply:
Ah, that's a good point :+1: I'll update